packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -309,6 +314,10 @@ const BlockInspectorSingleBlock = ( {
 							</div>
 						</>
 					) }
+
+					{ isNavigationMode && (
```

Comment:
Basically what I'm trying to achieve here is a slot for tools that should only appear in write mode.